public/app/features/alerting/unified/components/rule-editor/notificaton-preview/NotificationPreview.tsx

```diff
@@ -0,0 +1,155 @@
+import { css } from '@emotion/css';
+import { compact } from 'lodash';
+import React, { lazy, Suspense, useEffect, useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Button, LoadingPlaceholder, useStyles2 } from '@grafana/ui';
+
+import { Stack } from '../../../../../../plugins/datasource/parca/QueryEditor/Stack';
+import { AlertQuery, Labels } from '../../../../../../types/unified-alerting-dto';
+import { alertRuleApi } from '../../../api/alertRuleApi';
+import { Folder } from '../RuleFolderPicker';
+
+import { useGetAlertManagersSourceNamesAndImage } from './useGetAlertManagersSourceNamesAndImage';
+
+const NotificationPreviewByAlertManager = lazy(() => import('./NotificationPreviewByAlertManager'));
+
+interface NotificationPreviewProps {
+  customLabels: Array<{
+    key: string;
+    value: string;
+  }>;
+  alertQueries: AlertQuery[];
+  condition: string;
+  folder: Folder;
+  alertName?: string;
+  alertUid?: string;
+}
+
+export const NOTIFICATION_PREVIEW_TITLE = 'Alert instance routing preview';
+
+export const NotificationPreview = ({
+  alertQueries,
+  customLabels,
+  condition,
+  folder,
+  alertName,
+  alertUid,
+}: NotificationPreviewProps) => {
+  const styles = useStyles2(getStyles);
+  // potential instances are the instances that are going to be routed to the notification policies
+  const [potentialInstances, setPotentialInstances] = useState<Labels[]>([]);
```

Comment:
ok!, I created a `AlertInstance` type for it instead of using `Labels`. PreviewResponse is not entirely typed as we are only using part of it. I was searching for this type to be defined somewhere else but i couldn't find anything.